### ISSUE TITLE
Steel is too cheap

### ISSUE THREAD

**Issue by LemADEC** [BuiltBrokenModding/VoltzEngine#93]
Minecraft Version:
- 1.7.10
Forge Version:
- n/a
VoltzEngine Version:
- VoltzEngine-1.7.10-1.9.13b270

Explain issue in detail:
Steel is made from smelting iron ingots while it should consume some coal.
Assuming we don't have easy grinder, it would be something like:
4 charcoal + 1 iron ingot makes 1 Steel dust
3 coal + 1 iron ingot makes 1 Steel dust
3 charcoal/coal + 1 iron dust makes 1 Steel dust

**Comment by DarkGuardsman** [comment]
Shapeless with 3-4 carbon + 1 iron will be the likely recipe. Though I need to do some research before commiting to anything. However, I will implement the first one for the time being. As well with configs to change recipes :P

**Comment by LemADEC** [comment]
We need something out,  can always update later

**Comment by DarkGuardsman** [comment]
Yep, just wanted to note it will not stick around forever. Thinking of doing at least 4 methods to make steel. With each balanced towards a different gameplay style. The recipe we are doing is close to an old clay tube style. In which you would wrap the iron in charcoal powder. Then wrap it in clay in order to turn the iron into steel. The cool thing about this method is it works on just about any metal shape. So we could recycle it for the armory mod to do swords.
